pages/contribute/editors_checklist.md

```diff
@@ -19,9 +19,9 @@ summary: Checklist for editors before approving and merging a pull request (PR).
    * `resources`
 5. Make sure that listed tools or resources are tagged in the text with the [correct snippet](tool_resource_update) + that its metadata is described in the [tool_and_resource_list.yml](https://github.com/elixir-europe/rdmkit/blob/master/_data/tool_and_resource_list.yml) file. 
 6. Check if relevant recipes on FAIR Cookbook can be linked to the RDMkit page that is being added/changed ([Linking from RDMkit to FAIR Cookbook](editorial_board_guide#linking-from-rdmkit-to-fair-cookbook)).
-7. The content is conform to RDMkit scope, [style](style_guide) and templates.
+7. The content conforms to RDMkit scope, [style](style_guide) and templates.
```

Comment:
> The content conforms to RDMkit scope, ...

The content conforms to the RDMkit's scope,...

I think it needs a possessive.